packages/components/src/GuidanceBlock/GuidanceBlock.module.scss

```diff
@@ -0,0 +1,326 @@
+@import "~@kaizen/design-tokens/sass/shadow";
+@import "~@kaizen/design-tokens/sass/border";
+@import "~@kaizen/design-tokens/sass/layout";
+@import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/design-tokens/sass/animation";
+@import "~@kaizen/design-tokens/sass/spacing";
+@import "~@kaizen/component-library/styles/responsive";
```

Comment:
Migrate the responsive styles from `component-library` to `components/styles/legacy`